Geometry.cs

```diff
@@ -144,14 +158,46 @@ class Drawing {
 
    public Bound2 Bound {
       get {
-         if (mBound.IsEmpty) mBound = new (Polys.Select (a => a.Bound));
+         if (mBound.IsEmpty) mBound = new (ConvexHull);
          return mBound;
       }
    }
    Bound2 mBound;
 
-   public Bound2 GetBound (Matrix2 xfm) 
-      => new Bound2 (Polys.SelectMany (a => a.Pts.Select (p => p * xfm)));
+   IReadOnlyList<Point2> ConvexHull { 
+      get {
+         if (!mConvexHull.Any ()) ComputeConvexHull (Polys.SelectMany (a => a.Pts));
+         return mConvexHull;
+      } 
+   }
+   List<Point2> mConvexHull = new ();

```

Comment:
As soon as a Drawing is created, mConvexHull is instantiated and set to an empty list. If I never read the ConvexHull property, this is a wasted initialization. Can you have this initialized to null and actually created only the first time it is read?

Reply:
mConvexHull is now instantiated only when ConvexHull property is read